account/manager.go

```diff
@@ -165,10 +166,44 @@ func actionTxLabel(account *Account, action Action, isExpirySpend bool,
 	if err != nil {
 		log.Errorf("Internal error: failed to serialize json "+
 			"from %v: %v", label, err)
-		return fmt.Sprintf(prefixTag, action)
+		return fmt.Sprintf(txLabelPrefixTag, action)
+	}
+
+	return fmt.Sprintf(txLabelPrefixTag, labelJson)
+}
+
+// IsPoolTx returns true if the given transaction is related to
+// pool.
+func IsPoolTx(tx *lnrpc.Transaction) bool {
+	return strings.HasPrefix(tx.Label, txLabelPrefixTag)
+}
+
+// ParseTxLabel parses and returns data fields stored in a given account
+// transaction label.
+func ParseTxLabel(label string) (*AccountTxLabel, error) {
+	prefix := fmt.Sprintf(txLabelPrefixTag, "")
+	label = strings.TrimPrefix(label, prefix)
+
+	var data TxLabel
```

Comment:
The data in the label currently isn't written as `TxLabel` but as `AccountTxLabel` instead. So this doesn't work (returns an empty account). Did we decide to put the `"account":{}` level into the label or not? I don't remember.

Reply:
I think we did decide on `"account":{}` as far as I can tell from my notes here: https://github.com/lightninglabs/pool/issues/397